src/events.zig

```diff
@@ -254,8 +257,8 @@ pub const Type = enum(C.SDL_EventType) {
     // WindowLeaveFullscreen,
     // WindowDestroyed,
     // WindowHdrStateChanged,
-    // KeyDown,
-    // KeyUp,
+    key_down = C.SDL_EVENT_KEY_DOWN,
```

Comment:
Remember to have doc comments consistent with the SDL documentation: https://wiki.libsdl.org/SDL3/SDL_EventType